.github/workflows/image-promote.yml

```diff
@@ -1,11 +1,17 @@
-name: Sync workspace-full image tag from GAR to Dockerhub
+name: Sync images from GAR to Dockerhub
 
 on:
-  workflow_dispatch:
-    inputs:
-      tag:
-        description: 'worksapce-full image tag in GAR'
-        required: true
+  schedule:
```

Comment:
Let's talk about schedule, I'm not sure that we want to trigger every day just yet.

Reply:
Changed this to weekly. This would also mean that sync will take at least 1 week. Sure, let us discuss this offline